stubs/D3DShot/METADATA.toml

```diff
@@ -0,0 +1,5 @@
+version = "0.1.*"
+requires = ["types-Pillow"]
+
+[tool.stubtest]
+skip = true
```

Comment:
`skip = true` should only be specified under exceptional circumstances (which this is) -- we should add a comment to say why we're disabling stubtest here

```suggestion
# The library only works on Windows; we currently only run stubtest on Ubuntu for third-party stubs in CI.
# See #8660
skip = true
```